gax/src/test/java/com/google/api/gax/retrying/FailingCallable.java

```diff
@@ -48,19 +49,26 @@
           .build();
 
   private AtomicInteger attemptsCount = new AtomicInteger(0);
+  private final ApiTracer tracer;
   private final int expectedFailuresCount;
   private final String result;
 
-  FailingCallable(int expectedFailuresCount, String result) {
+  FailingCallable(ApiTracer tracer, int expectedFailuresCount, String result) {
```

Comment:
Super-minor, but I think `ApiTracer` should be the last parameter in the constructor, not the first one (usually ordering of arguments is by importance, and tracer is by definition optional, also it was added later historically).